tests/run-ci-benchmarks.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+JSON_FILE="benchmark-data.json"
+
+cat << EOF >> ${JSON_FILE}
+[
+EOF
+
+DIRNAME=$(dirname "$0")
+
+SUCC=$(find "${DIRNAME}"/cn -name '*.c' | grep -v '\.error\.c')
```

Comment:
```bash
SUCC=($(find "${DIRNAME}"/cn -name '*.c' | grep -v '\.error\.c'))
```
But measuring the time taken for failing benchmarks would be useful too right?

Reply:
Makes sense. I've added failing tests.